libbeat/docs/shared-kibana-config.asciidoc

```diff
@@ -85,6 +85,13 @@ An HTTP path prefix that is prepended to the HTTP API calls. This is useful for
 the cases where Kibana listens behind an HTTP reverse proxy that exports the API
 under a custom prefix.
 
+[float]
+[[kibana-space-id-option]]
+==== `setup.kibana.space.id`
+
+The Kibana Space ID to use. If specified, {beatname_uc} will load Kibana assets
+into this Kibana Space. Omit this option to use the Default Kibana Space.
```

Comment:
```suggestion
The Kibana space ID to use. If specified, {beatname_uc} loads Kibana assets
into this Kibana space. Omit this option to use the default space.
```

Reply:
My edits are based on the capitalization/style conventions used in the Kibana topic about spaces: https://www.elastic.co/guide/en/kibana/7.6/xpack-spaces.html

Do you think it's worth adding a link to the docs about Kibana spaces? If so, you might want to make Kibana space an active link: {kibana-ref}/xpack-spaces.html[Kibana space]